app/src/main/kotlin/com/simplemobiletools/smsmessenger/activities/MainActivity.kt

```diff
@@ -627,6 +627,7 @@ class MainActivity : SimpleActivity() {
             Intent(Intent.ACTION_GET_CONTENT).apply {
                 addCategory(Intent.CATEGORY_OPENABLE)
                 type = EXPORT_MIME_TYPE
+                putExtra(Intent.EXTRA_MIME_TYPES, arrayOf(EXPORT_MIME_TYPE, XML_IMPORT_MIME_TYPE))
```

Comment:
guess EXPORT_MIME_TYPE should be renamed now to reflect json. And maybe we should allow importing .txt files too, it is often used even while the actual content is in .json or .xml. The reporter provided a .txt too.

Reply:
I think they mentioned that they changed the extension to `.txt` to be able to upload it to GitHub, but I can add support for TXT too, since I already had to check the first line of file to detect if it has XML contents.

Reply:
right, he renamed it manually, didnt notice. But ye, such files can often be in .txt too.

Reply:
Alright, I will add support for txt too then